man/systemd-coredump.xml

```diff
@@ -52,14 +52,26 @@
 
   <refsynopsisdiv>
     <para><filename>/usr/lib/systemd/systemd-coredump</filename></para>
+    <para><filename>/usr/lib/systemd/systemd-coredump</filename> <option>--backtrace</option></para>
     <para><filename>systemd-coredump@.service</filename></para>
     <para><filename>systemd-coredump.socket</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
     <title>Description</title>
-    <para><command>systemd-coredump</command> is a system service that can acquire core dumps
-    from the kernel and handle them in various ways.</para>
+    <para><command>systemd-coredump@.service</command> is a system service that can acquire core
```

Comment:
for unit names we so far used `<filename></filename>`, as they usually map 1:1 to filenames...